one_fm/api/doc_methods/oauth_bearer_token.py

```diff
@@ -0,0 +1,14 @@
+import frappe
+from frappe.integrations.oauth2 import revoke_token
+
+def revoke_and_delete_existing_tokens(doc, event):
+    """
+        Revokes and deletes all the existing tokens for the target user.
+    """
+    user_existing_tokens = frappe.get_all('OAuth Bearer Token', filters={'user': doc.user}, fields=['name'])
```

Comment:
@ibtesam-fm get_all filters should be 
1. Client id
2. Status (get only Active tokens)
3. Additionally can even pass name filter to not include current token.

Reply:
Sure I will do that. But we are deleting the tokens after revoking so can we neglect status filters because that seems to be useless I think?